gulp/config.js

```diff
@@ -0,0 +1,51 @@
+var dest = './doc/public',
+    src = './pattern-library',
+    siteDir = './_site',
+    docDir = './doc';
+
+module.exports = {
+    documentation: {
+        testing: {
+            sources: [
+                './src/js/utils/spec-helpers/*.js'
+            ],
+            output: './doc/_testing',
+            viewClass: 'testing'
+        },
+        utilities: {
+            sources: [
+                './src/js/utils/*.js'
+            ],
+            output: './doc/_utilities',
+            viewClass: 'utility'
+        },
+        views: {
+            sources: [
+                './src/js/disclosure/*.js',
+                './src/js/dropdown-menu/*.js',
+                './src/js/pagination/*.js'
```

Comment:
Yep, that would make more sense. It will have to filter out utils so that it doesn't duplicate all of those files.